TS Interface Error for `src` attribute

Hi there! I just tried to upgrade this package from `1.3.1` to `2.3.0` to take advantage of the new TS declaration, but realized that I'm getting a TS error regarding the `src` attribute. Specifically the error is: 

```
Interface 'ImgProps' incorrectly extends interface 'Pick<DetailedHTMLProps<ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement>
...

 Types of property 'src' are incompatible.
    Type 'string | string[]' is not assignable to type 'string'.
      Type 'string[]' is not assignable to type 'string'.

4     export interface ImgProps extends React.ComponentPropsWithoutRef<'img'> {
                       ~~~~~~~~


Found 1 error.
```

Updating the TS definition to `export interface ImgProps extends Omit<React.ComponentPropsWithoutRef<'img'>, 'src'> { ... }` seems to fix the issue, since it looks like the underlying `src` DOM attribute is expecting a string. 

Curious about a good path forward. Are others seeing this? If so, happy to put up a PR to fix. Let me know -- thanks!

Same issue:
`/usr/src/app/client/node_modules/react-image/react-image.d.ts
(4,22): error TS2430: Interface 'ImgProps' incorrectly extends interface 'Pick<DetailedHTMLProps<ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement>, "style" | "title" | ... 261 more ... | "useMap">'.
  Types of property 'src' are incompatible.
    Type 'string | string[] | undefined' is not assignable to type 'string | undefined'.
      Type 'string[]' is not assignable to type 'string'.`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,6 @@
+# 2.4.0
+- fix: TS Interface Error for 'src' attribute. Related to issue: #260
+
 # 2.3.0
 
 - fix: typescript declarations

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "react-image",
-  "version": "2.3.0",
+  "version": "2.4.0",
   "description": "React Image is an <img> tag replacement for react, featuring preloader and multiple image fallback support",
   "scripts": {
     "build": "NODE_ENV=production rollup -c && for i in cjs esm umd; do cp src/*test.js $i; done",

--- a/react-image.d.ts
+++ b/react-image.d.ts
@@ -1,7 +1,7 @@
 declare module 'react-image' {
     import * as React from 'react'
   
-    export interface ImgProps extends React.ComponentPropsWithoutRef<'img'> {
+    export interface ImgProps extends Omit<React.ComponentPropsWithoutRef<'img'>, 'src'> {
       src?: string | string[]
       loader?: JSX.Element
       unloader?: JSX.Element

